### PR TITLE
Support showing only jobnames on non-successful jobs

### PR DESCRIPTION
**Pull request by norrs**

### CODE REVIEW

**Comment by timoschwarzer** [comment]
- I'd rename the config key to `showJobsNameOnlyOn`.
- I'd always suggest a use case in the description for more complicated configuration items:

```
// Only show job names for specific job statuses.
// Valid values are: 'created', 'pending', 'running', 'failed','success', 'canceled', 'skipped' or 'manual'.
// This is an example that shows job names only for jobs with a non-success status:
```

File: CONFIGURATION.md [hunk 1]
```diff
@@ -36,6 +36,19 @@ Your configuration overrides the default configuration, which can be found
   // Can be: 'icon', 'name', 'iconAndName'
   "showJobs": "icon",
 
+  // If entries in showJobsNameOnStatus, will only show names for job statuses matching any of the names given.
+  // Array can have entries of the following: 'created', 'pending', 'running', 'failed','success', 'canceled', 'skipped' or 'manual'.
+  // See https://docs.gitlab.com/ee/api/jobs.html#list-project-jobs and available scopes.
+  // Example below is to only show job names for non-success jobs.
```
Comment (norrs) [inline CONFIGURATION.md]: @timoschwarzer We can reword this , as this is our use case, to only show job names on non-successful jobs. If a job is green, we don't really care what the success is.

Comment (timoschwarzer) [reply CONFIGURATION.md]: Feel free to use my proposed text :)

**Comment by norrs** [comment]
There we go @timoschwarzer , changed accordingly to feedback.

**Comment by timoschwarzer** [comment]
Thank you!

**Pull request merged**